internal/apis/certmanager/types_certificate.go

```diff
@@ -171,6 +171,12 @@ type CertificateSpec struct {
 	// +optional
 	RenewBeforePercentage *int32
 
+	// `renewal` allows configuration of how your certificate is renewed. If the policy mentioned is
+	// `RenewBefore` then the controller respects `renewBefore` and `renewBeforePercentage`. If the policy is
+	// `Disabled` or `EarliestWindow` then it ignores those fields.
```

Comment:
```suggestion
	// `renewal` allows configuration of how your certificate is renewed. If the policy mentioned is
	// `RenewBefore` then the controller respects `renewBefore` and `renewBeforePercentage`.
```

`EarliestWindow` is not (yet) a valid policy. And it's pretty obvious that renewal fields are ignored if the renewal is `Disabled`.